modules/nifti/src/niftireader.cpp

```diff
@@ -112,8 +156,17 @@ const DataFormatBase* NiftiReader::niftiDataTypeToInviwoDataFormat(int niftiData
             return nullptr;
     }
     // clang-format on
-}
```

Comment:
`DT_RGB24` and `DT_RGBA32` should also only set type and precision.

Reply:
I think that components will be 1 for these types, which do not translate to components in our DataFormat

Reply:
I do not have a data set to test this format though.

Reply:
I see